app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -232,7 +232,13 @@ export default class List extends PureComponent {
             closeButton: this.closeButton,
         };
 
-        showModal(screen, title, passProps);
+        const options = Platform.select({
+            ios: {
+                modalPresentationStyle: 'pageSheet',
```

Comment:
A bit of self-review:
I was wondering if it would be better to create this as a default prop that can just be referenced by all 4 modals in this component. It may actually not make much of a difference though

Reply:
I think you can set this as a default option in `showModal` in the navigation action